src/commands/release.ts

```diff
@@ -12,7 +12,7 @@ async function release () {
   const repoPath = await Utils.repository.getPath (),
         version = await Utils.repository.getVersion ( repoPath );
 
-  if ( !repoPath || !version ) return Utils.exit ( '[release] Unsupported repository' );
+  if ( !Config.release.github.owner || !Config.release.github.repo || !repoPath || !version ) return Utils.exit ( '[release] Unsupported repository' );
```

Comment:
Actually we always need a repoPath because we need that for retrieving the changelog here: https://github.com/fabiospampinato/bump/blob/0cfa315225f8c2877619a851d9fee0555e4a38bd/src/providers/release/github.ts#L42.

So this line shouldn't be changed at all I think.

Reply:
Plus there's no reason to exit if custom owner or repo haven't been provided.

Reply:
@fabiospampinato Resolved